dokka-runners/dokka-gradle-plugin/src/main/kotlin/DokkaExtension.kt

```diff
@@ -131,12 +131,13 @@ constructor(
 
     interface Versions : ExtensionAware {
 
-        /** Default version used for Dokka dependencies */
-        val jetbrainsDokka: Property<String>
-        val jetbrainsMarkdown: Property<String>
-        val freemarker: Property<String>
-        val kotlinxHtml: Property<String>
-        val kotlinxCoroutines: Property<String>
+        /**
+         * The default version of Dokka dependencies that are used at runtime during generation.
+         *
+         * This value defaults to the current Dokka Gradle Plugin version, but can be overridden
+         * if you want to use a newer or older version of Dokka at runtime.
+         */
+        val dokkaEngine: Property<String>
```

Comment:
why not to move this on extension level and call just `dokkaEngineVersion`/`engineVersion` - similar to how KGP has `compilerVersion` when using `build-tools-api`?

Not sure we need `Versions` interface anymore

Reply:
Sure, I'll move it to the top level.

The reason I wanted to keep it nested inside a `versions {}` block was to avoid having too many options in the top level of `dokka {}`. I think keeping the lower-level options nested helps indicate that they are more complicated and advanced. Although I agree that a `versions {}` block seems pretty pointless without other versions, and it's unlikely that we'll be adding other versions.

tbh I think the top level `dokka {}` DSL is pretty disorganised and could do with a review, but it's hard to re-organise it without becoming too disconnected from the underlying Dokka configuration.

Anyway, we can revisit this later if needed.